uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -77,4 +77,41 @@ Feature: Greengrass V2 LogManager
         Then the Greengrass deployment is COMPLETED on the device after 5 minutes
         Then I verify the aws.greengrass.LogManager component is RUNNING using the greengrass-cli
         And I verify that it created a log group for component type UserComponent for component UserComponentA, with streams within 120 seconds in CloudWatch
+        And I verify the rotated files are deleted except for the active log file for component UserComponentA
```

Comment:
This is not what we should have here - The current main branch has this step 

```
        And I verify the rotated files are deleted and that the active log file is present for component UserComponentA
```

Instead of this one you have here for LogManager-1-T2